packages/runtime/src/builder/serialization/control-serialization.ts

```diff
@@ -1044,7 +1044,18 @@ export function deserializeControls(
 ): Record<string, DeserializedControl> {
   return Object.entries(serializedControls).reduce(
     (deserializedControls, [key, serializedControl]) => {
-      return { ...deserializedControls, [key]: deserializeControl(serializedControl) }
+      try {
+        const deserializedControl = deserializeControl(serializedControl)
+        return { ...deserializedControls, [key]: deserializedControl }
+      } catch (err: unknown) {
+        if (err instanceof Error) {
+          console.error(
```

Comment:
Will this still get reported to Sentry?

Reply:
It will not, and I'm not sure that we'd want to? I'd imagine that reporting to Sentry for this would raise a lot of noise for what are effectively user errors.

Reply:
I think we want it to be reported, it's unclear to me whether all or even most of these are going to be user errors, especially after we relax the schema to enable built-in coercions.